src/App.vue

```diff
@@ -458,7 +458,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         // If Summary Mode: Check validity, save and file else move into summary mode.
         this.isSummaryMode
           ? await this.validateApp()
-          : this.setSummaryMode(true)
+          : this.scrollToTop(document.getElementById('app')); this.setSummaryMode(true)
```

Comment:
Do both statements on this line get executed? It's not clear from the syntax.

I think this would be better within an actual if-then-else with curly braces.

Reply:
They do, because of the semi-colon. Javascript madness!

Will refactor for readability tho!

Reply:
Updated in next commit!